api-specificatie/openapi.yaml

```diff
@@ -1284,19 +1276,19 @@ components:
           $ref: '#/components/schemas/OpenbareRuimteMogelijkOnjuist'
     OpenbareRuimteMogelijkOnjuist:
         type: object
-        properties: 
+        description: Wanneer hier een property is opgenomen met de waarde true, dan is de waarde van het gelijknamige property in de resource mogelijk onjuist. De juistheid van dit gegeven wordt op dit moment onderzocht. Onder property toelichting wordt toegelicht wat er mogelijk onjuist is aan het betreffende gegeven.
+        properties:
           naam:
             type: boolean
-            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
-          type:
```

Comment:
Is het de bedoeling dat het type openbare ruimte hier verdwijnt? In openbare ruimte zelf zit nog wel het type, deze lijkt me dan ook nodig.

Reply:
goed opgemerkt, dank je. Die was er op de een of andere manier tussenuit gevallen. Heb het weer toegevoegd